src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAnAppHost.cs

```diff
@@ -197,11 +197,11 @@ private string PrepareAppHostMockFile(TestDirectory testDirectory, Action<byte[]
             return filePath;
         }
 
-        private const int _subsystemOffset = 0xF0 + 0x5C;
+        private const int SubsystemOffset = 0xF0 + 0x5C;
 
         // This is a dump of first 350 bytes of a windows apphost.exe
         // This includes the PE header and part of the Optional header
-        private static byte[] _windowsFileHeader = new byte[] {
+        private static byte[] WindowsFileHeader = new byte[] {
```

Comment:
This one is technically not a constant. I would have used s_windowsFileHeader from my corefx upbringing, but we don't seem to be consistent on that and I think our editorconfig was tweaked to use just _ even for static fields. 

I don't care enough about this to block the commit, though. Seems we should make a call and fix all the static fields up separately.

Reply:
Commented on https://github.com/dotnet/sdk/pull/1589